dependencydiff/dependencydiff_test.go

```diff
@@ -40,8 +40,8 @@ func Test_fetchRawDependencyDiffData(t *testing.T) {
 				ctx:       context.Background(),
 				ownerName: "no_such_owner",
```

Comment:
in a follow-up PR, it would be useful to have better tests for getScorecardCheckResults(). There's a bunch of logic in there that would be nice to have tests for. It may require some code-refactoring.